src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -196,53 +198,36 @@ private DetailsView AddDetailsView()
 
         private void InitializeResourceStreams()
         {
-            var rootnamespace = "Dynamo.LibraryUI.web.";
-            var resourceNames = Assembly.GetExecutingAssembly()
-            .GetManifestResourceNames();
-            foreach (var resource in resourceNames)
+            resourceFactory = new ResourceHandlerFactory();
+            resourceFactory.RegisterProvider("/dist/v0.0.1", 
+                new DllResourceProvider() { BaseUrl = "http://localhost/dist/v0.0.1",
+                    RootNamespace = "Dynamo.LibraryUI.web.library" });
+
             {
-                if (!resource.StartsWith(rootnamespace))
-                    continue;
+                var url = "http://localhost/library.html";
+                var resource = "Dynamo.LibraryUI.web.library.library.html";
+                var stream = LoadResource(resource);
+                resourceFactory.RegisterHandler(url, ResourceHandler.FromStream(stream));
+            }
 
-                var url = resource.Replace(rootnamespace, "");
-                if (url.StartsWith("dist."))
-                {
-                    url = url.Replace("dist.", "dist/");
-                    url = url.Replace("/v0._0._1.", "/v0.0.1/");
-                    url = url.Replace("/resources.", "/resources/");
-                    //url = url.Replace("/icons.", "/icons/");
-                    //url = url.Replace(".font_awesome_4._7._0.", "/font-awesome-4.7.0/");
-                    //url = url.Replace("/fonts.", "/fonts/");
-                    //url = url.Replace("less.", "less/");
-                    //url = url.Replace("css.", "css/");
-                }
-                
-                if (url.EndsWith(".json"))
-                {
-                    url = url.Replace(".json", "");
-                }
-                var r = LoadResource(resource);
```

Comment:
LoadResource has been used to get the resource stream and a resource handler is created using the stream. Please see line 209/216/223.